databases/week1/db-week1-hw.sql

```diff
@@ -0,0 +1,58 @@
+1. Find out how many tasks are in the task table
+SELECT COUNT(*) FROM task;
+
+2. Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(*) FROM task WHERE due_date IS NULL;
+
+
+3. Find all the tasks that are marked as done
+SELECT task.id, task.title, status.name, status.id FROM task, status where status.name LIKE "Done" 
+AND task.status_id=status.id;
```

Comment:
This could be easier & fewer code that gives the same output. This may applies to nr.4 as well.
```
select * 
from task 
join status on task.status_id = status.id 
where status.name  = 'Done';
```